Add types of nodes, and node implementations. (#941)
ipfs/ipfs-docs: docs/concepts/nodes.md

---
title: "Nodes"

> description: "Computers running IPFS are called nodes. Nodes are the most imporant aspect of IPFS - without nodes running the IPFS daemon, there would be no such thing as IPFS. This page discusses what nodes are, current IPFS implementations, and the types of services different nodes can offer."

Reviewer: ```suggestion
description: "Participants in the IPFS network are called nodes. Nodes are the most important aspect of IPFS - without nodes running the IPFS daemon, there would be no  IPFS network. This page discusses what nodes are, current IPFS implementations, and the types of services different nodes can offer."
```

Reviewer: there *would* be such a thing as IPFS - the IPFS protocol - even if no nodes were running it. A protocol doesnt require running implementations in order to be valid. However the network does require active nodes - so rephrased

---

# Nodes

> Computers running IPFS are called _nodes_. Nodes are the most crucial aspect of IPFS - without nodes running the IPFS daemon, there would be no such thing as IPFS in the first place.

Reviewer: ```suggestion
Participants in the IPFS network are called _nodes_. Nodes are the most crucial aspect of IPFS - without nodes running the IPFS daemon, there would be no IPFS Network.
```
